src/registrar/management/commands/utility/terminal_helper.py

```diff
@@ -317,9 +373,8 @@ def print_conditional(
                 case _:
                     logger.info(print_statement)
 
-    # TODO - "info_to_inspect" should be refactored to "prompt_message"
```

Comment:
This seemed like an easy thing to just throw in, let me know if more work was intended here.